configs/prod/roles/vulnerability.json

```diff
@@ -2,10 +2,11 @@
   "roles": [
     {
       "name": "Vulnerability administrator",
-      "description": "Perform any available operation against any Vulnerability resource.",
+      "description": "Perform any available operation on any Vulnerability resource.",
       "system": true,
-      "platform_default": true,
-      "version": 10,
+      "platform_default": false,
+      "admin_default": true,
```

Comment:
this PR should contain only name + description changes, please if you want to change `platform_default` and `admin_default` flags, please create a separate PR, thank you